app/signals/settings/base.py

```diff
@@ -291,6 +291,7 @@ def is_super_user(user):
 EMAIL_HOST_PASSWORD = os.getenv('EMAIL_HOST_PASSWORD')
 EMAIL_PORT = os.getenv('EMAIL_PORT', 465)  # 465 fort SSL 587 for TLS
 EMAIL_USE_TLS = os.getenv('EMAIL_USE_TLS', False) in TRUE_VALUES
+EMAIL_TIMEOUT = os.getenv('EMAIL_TIMEOUT', 5)
```

Comment:
Normally, it's preferable to use the existing value as the default when setting environment variables. Which should be `None` in this case, however `5` is a reasonable default to have. So this is not blocking.

Reply:
👍